### ISSUE TITLE
Requests for private resources cause refference error

### ISSUE THREAD

**Issue by michaelost** [samholmes/node-open-graph#9]
https://github.com/samholmes/node-open-graph/blob/master/index.js#L47
It should be res.statusCode instead of just statusCode. I could make such minor fix if I had permissions.

**Comment by ktrzeciaknubisa** [comment]
It is not just private resource. When I try to call og for the following public URL:

http://www.honda-ariesmotor.pl/pl/samochody/nasza-firma/kontakt.html

I get same error:

```bash
/.../node_modules/open-graph/index.js:47
				cb(new Error("Request failed with HTTP status code: "+statusCode));
				                                                      ^

ReferenceError: statusCode is not defined
    at Request._callback (/.../node_modules/open-graph/index.js:47:59)
```

**Comment by samholmes** [comment]
This has been officially fixed: #12

Sorry for the delay on fixing this. This is was dumb mistake on my part. Now it's gone.
